v4/source/system.yaml

```diff
@@ -737,6 +737,34 @@
 
             // GetOldClientLicense
             license, resp := Client.GetOldClientLicense()
+  /license/renewal:
+    get:
+      tags:
+        - system
+      summary: Request the license renewal link
+      description: >
+        Request the renewal link that would be used to start the license renewal process
+
+        __Minimum server version__: 5.32
+
+        ##### Permissions
+
+        Must have `manage_system` permission.
```

Comment:
Is this accurate? It seems we are checking the read about license permission. Should we fallback in this one instead?

Reply:
I've updated that to reflect the permission needed